python/mxnet/ndarray.py

```diff
@@ -303,6 +303,23 @@ def _copy_slice_to(self, axis, start, stop, target):
         assert isinstance(target, NDArray)
         return _internal._copy_slice_to(self, axis, start, stop, out=target)
 
+    def assign_slice_from(self, axis, start, stop, source):
```

Comment:
Do you mean creating a new op or are you referring to some existing op that I missed?